spec/liquid_tags/golang_tag_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+RSpec.describe GolangTag, type: :liquid_tag do
+  describe "#link" do
+    let(:valid_link) { "https://play.golang.org/p/HmnNoBf0p1z" }
+
+    def generate_tag(link)
+      Liquid::Template.register_tag("golang", GolangTag)
+      Liquid::Template.parse("{% golang #{link} %}")
+    end
+
+    it "accepts valid input" do
+      expect { generate_tag(valid_link) }.not_to raise_error
+    end
+
+    it "renders valid input" do
+      template = generate_tag(valid_link)
+      expected = 'src="true"'
```

Comment:
This needs to check that the the src attribute points to the valid link:

```suggestion
      expected = 'src="#{valid_link}"'
```